modules/extensions-spring-support/src/test/resources/models/petstore.json

```diff
@@ -18357,9 +18369,13 @@
       "name": "PartialPetAdoptionSource",
       "description": "",
       "modelProperties": {
-        "org.mule.runtime.extension.api.property.TypeResolversInformationModelProperty": {
+        "typeResolversInformation": {
```

Comment:
aren't you breaking BP with this?

Reply:
nop, this was added on 1.2.0 (current SNAPSHOT version) so we are still able to fix it